pipeline/raw/get_vax_data/main.py

```diff
@@ -9,24 +9,31 @@
 from epimargin.utils import mkdir
 from google.cloud import storage
 
-# cloud details 
+# cloud details
 bucket_name = "daily_pipeline"
 
+
+def download_vaccine_data_dropbox(dropbox_data_path):
```

Comment:
do we need this download here if we just directly download in the `exp.py` file?

Reply:
We actually don't. I just added this function more like for "completeness" (i.e., if we want to access this csv file from a bucket later like how we access the other csv files)